src/q6/telecom-ELEC1360/exam/2019/Juin/Majeure/telecom-ELEC1360-exam-2019-Juin-Majeure.tex

```diff
@@ -0,0 +1,65 @@
+\documentclass[fr]{../../../../../../eplexam}
+
+\hypertitle{Télécommunications}{6}{ELEC}{1360}{2019}{Juin}{Majeure}
+{Quentin Dessain}
+{Luc Vandendorpe}
+
+\section{Soit $X(t)$ un signal aléatoire à bande étroite B, faiblement stationnaire. On note sa densité spectrale de puissance par $\gamma_{X}(w)$ et ses composantes de Rice, extraites autour de la porteuse à pulsation $w_{0}$, $X_{1}(t)$ et $X_{2}(t)$.}
```

Comment:
Vu que ce paragraphe fait partie de l'énoncé au même titre que l'énumération du dessous, je ne suis pas convaincu que le mettre dans le `\section{}` (et donc en gras) soit approprié : on peut ne rien mettre dans le `\section{}`.
Ca reste mon avis ;-)
(Autre avantage de le mettre dehors : ça diminue la quantité de warning "Token not allowed in a PDF string" de hyperref.)

Reply:
C'est modifié ;)